src/main/java/org/fenixedu/academic/domain/dml/DynamicFieldDescriptor.java

```diff
@@ -186,13 +189,22 @@ public void moveUp() {
             return;
         }
 
+        if (findAtPosition(getDomainObjectClassName(), currentIndex - 1).getTag() != getTag()) {
```

Comment:
as we discussed, for now it's better to remove all these adjacent tags validations